pkg/kubecost/allocation.go

```diff
@@ -966,10 +965,8 @@ func (as *AllocationSet) AggregateBy(aggregateBy []string, options *AllocationAg
 			for _, idleAlloc := range idleSet.allocations {
 				// Only share idle if the idleKey matches; i.e. the allocation
 				// is from the same idleKey as the idle costs
-				iaIdleKey, err := idleAlloc.getIdleKey(options)
-				if err != nil {
-					return err
-				}
+				iaIdleKey, _ := idleAlloc.getIdleKey(options)
```

Comment:
Should we only swallow this error if we're doing idle by node? Are we sure that we want to alter the old behavior to support the new behavior?

Reply:
I am ignoring these because they imply a node without a cluster or a node without a provider ID which from what I can tell does not exist

Reply:
Hm. I'm not sure I agree that returning an error and failing the query is wrong here. (If this implies a node without a cluster, and that shouldn't be possible, then I think the only correct behavior upon encountering an error here is to return it.) But I don't want to obstruct. Can we at least log the error before moving on?

Reply:
added back in since it will not interfere with the feature